core/domain/fs_domain.py

```diff
@@ -320,7 +322,7 @@ def listdir(self, dir_name):
         for metadata_model in metadata_models:
             filepath = metadata_model.id
             if filepath.startswith(prefix):
-                # Because the path is /exploration/exp_id/assets/abc.png.
+                # Because the path is /entity/entity_id/assets/abc.png.
```

Comment:
Probably put {{ ... }} around the first two parts (or, perhaps better, `<...>` as is done below for the GCS comments)

Reply:
Done